.github/workflows/perf-runtime.yml

```diff
@@ -0,0 +1,147 @@
+name: Perf - Runtime
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+  benchmark:
+    strategy:
+      fail-fast: false
+      # `matrix` creates a job matrix (Cartesian product of possible values for each var)
+      # https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
+      matrix:
+        component:
+          - locale
+          - num-util
+          - uniset
+      # Ensure that commits when pushing GH pages to external has linear commit
+      # history w/o merges (to keep things simpler). Do this via max-parallel=1.
+      # Also, necessary b/c can't figure out how to use component-specific branches
+      # and coordinate between the benchmark action step and the GH pages action step.
+      max-parallel: 1
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Debug branches
+        run: echo "git branch" && git branch
+
+      # Download previous benchmark result from cache (if exists)
+      - name: Download/cache previous benchmark data
+        uses: actions/cache@v1
+        with:
+          path: dev
+          key: ${{ runner.os }}-benchmark
+
+      - run: mkdir -p dev/${{ matrix.component }}/bench/cache
+
+      - name: Debug bench info
+        run: echo "ls -l dev" && ls -l dev && echo "ls -l dev/${{ matrix.component }}/bench" && ls -l dev/${{ matrix.component }}/bench
+      # - name: Debug cache info
+      #   run: echo "ls -l dev" && ls -l dev && echo "ls -l dev/${{ matrix.component }}/bench/cache" && ls -l dev/${{ matrix.component }}/bench/cache
+      #- run: mkdir -p dev/${{ matrix.component }}/bench/cache
+
+      - name: Try on a particular component
+        run: cd components/${{ matrix.component }}
```

Comment:
what's the goal here?

Reply:
I think this is cruft from changing debugging steps.